source_sdk/entity/entity.h

```diff
@@ -69,4 +69,42 @@ struct player_info_t
 	unsigned char files_downloaded;
 };
 
+// usefull for using with get_ent_flags()
+// EX:  bool on_ground = get_ent_flags(localplayer) & FL_ONGROUND;
```

Comment:
You can totally move this somewhere else you think it's better for the project. In my view this fits here.

Reply:
I think it makes sense here